File: codes/prime.py
# PRIME Function
def prime(n):
  for i in range(2,n):
    if n%i == 0:
      return False
  return True

if __name__ == "__main__":
  # input an integer
  n = int(input("Enter an integer n: "))
  if prime(n):
    print("Prime")
  else:
    print("Not Prime")

File: codes/prime.cpp
#include <iostream>
using namespace std;

// PRIME Function
bool prime(int n) {
  for (int i = 2; i < n; i++) {
    if (n%i == 0) return false;
  }
  return true;
}

int main() {
  // input an integer
  int n;
  cout << "Enter an integer n: ";
  cin >> n;
  if (prime(n)) cout << "Prime" << endl;
  else cout << "Not Prime" << endl;
}